torch/_functorch/partitioners.py

```diff
@@ -888,6 +903,86 @@ def get_node_weight(node) -> int:
         for user in node.users:
             nx_graph.add_edge(node.name + "_out", user.name + "_in", capacity=math.inf)
 
+    # todo(chilli): This is the most questionable of the 3 heuristics for banning recompute.
+    # I would look more into it but I'm out of time for now
```

Comment:
you can prob leave this sentence out haha. that's what are all todos are for lol